src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int UPPER_BOUND = 10;
+    private static final int LOWER_BOUND = 1;
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        int number1 = random.nextInt(UPPER_BOUND - LOWER_BOUND + 1) + LOWER_BOUND;
+        int number2 = random.nextInt(UPPER_BOUND - LOWER_BOUND + 1) + LOWER_BOUND;
```

Comment:
```suggestion
        int firstNumber = random.nextInt(UPPER_BOUND - LOWER_BOUND + 1) + LOWER_BOUND;
        int secondNumber = random.nextInt(UPPER_BOUND - LOWER_BOUND + 1) + LOWER_BOUND;
```
please avoid numbers in variable names